request.go

```diff
@@ -65,7 +65,7 @@ func prepareAuthRequest(requestID string, username string, password string) requ
 	req := request{}
 	req.RequestID = requestID
 	req.Op = "authentication"
-	req.Processor = "trasversal"
+	req.Processor = "traversal"
```

Comment:
how did this work beforehand? ... is the processor ignored for the auth operation?

Reply:
There is no test that uses auth, so I guess it is ignored.